python/kserve/kserve/model_server.py

```diff
@@ -124,14 +124,21 @@ def __init__(self, http_port: int = args.http_port,
         self.access_log_format = access_log_format
 
     def start(self, models: Union[List[Model], Dict[str, Deployment]]) -> None:
-        if isinstance(models, list):
+        if not models:
+            raise RuntimeError("Model is not provided")
+        elif isinstance(models, list):
+            at_least_one_model_ready = False
             for model in models:
                 if isinstance(model, Model):
+                    if model.ready:
+                        at_least_one_model_ready = True
                     self.register_model(model)
                     # pass whether to log request latency into the model
                     model.enable_latency_logging = self.enable_latency_logging
                 else:
                     raise RuntimeError("Model type should be 'Model'")
+            if not at_least_one_model_ready:
+                raise RuntimeError("No ready Model")
```

Comment:
We can use [ModelNotReady error](https://github.com/kserve/kserve/blob/27f13d02966fe7ba40fa69afcab2f861819db134/python/kserve/kserve/errors.py#L80) from kserve.

Reply:
Due to the ModelNotReady not being able to accept input of List type, I have created a class named NoModelReady to print an error message.  please help me check, Thanks!